atlasdb-client-protobufs/src/main/java/com/palantir/atlasdb/protos/generated/TableMetadataPersistence.java

```diff
@@ -1,5 +1,5 @@
 // Generated by the protocol buffer compiler.  DO NOT EDIT!
-// source: proto/com/palantir/atlasdb/protos/TableMetadataPersistence.proto
+// source: main/proto/com/palantir/atlasdb/protos/TableMetadataPersistence.proto
```

Comment:
It seems the protos depend on the directory from which you compile them! Had no idea this was a thing :)